### DISCUSSION TITLE
Metrics from logs json parser issue

### DISCUSSION THREAD

**Discussion by nthomaslife360** [vectordotdev/vector#21095] [Q&A]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Problem

Hi,

I have am trying to parse a below event to Vector via s3 and trying to fetch metrics out of it.

#sample json event

<img width="594" alt="image" src="https://github.com/user-attachments/assets/ad370241-c241-43f5-931d-6d50a554e193">

I'm uploading this event to s3 and trying to get `path` and `request_count`.

It is getting ingested BUT while testing it by printing it out to internal console logs before creating metrics can see that each event `message` is getting parsed as single event, attached snippet below

![Untitled design](https://github.com/user-attachments/assets/9e12e5af-71c1-4155-97e3-ac4f62736cf4)

### Configuration

```text
testing_te_file_remap:
        type: remap
        inputs:
          - route_s3_normalized.mobile_logs
        source: |-
           . = parse_json!(.message)
```

I want this to be parsed as single event like below 
```

{"bucket":"life360-mobile-logs-dev","request_type":"mobile_logs","source_type":"aws_s3","message":"      

 {
    "path": "/",
    "request_count": 14,
    "average_duration": 1138.4285714285713,
    "total_data_transferred": 550994,
    "methods_distribution": [
      {
        "method": "POST",
        "count": 14
      }
    ]
  }
,"region":"us-east-1","object":"33.json"}
```

### Version

"0.36.0-distroless-libc

### Example Data

Attached above

**Comment by jszwedko** [comment]
I converted this to a discussion. We are also discussing this in Discord: https://discord.com/channels/742820443487993987/1274053459213291611/1274058078144303236
